src/components/Checkbox/Checkbox.tsx

```diff
@@ -88,7 +88,7 @@ export const Checkbox = ({
     multiline={!!subtitle}
     onClick={() => onChange(!checked)}
   >
-    <AminoCheckbox checked={checked}>
+    <AminoCheckbox checked={checked} id="amino-checkbox">
```

Comment:
An id needs to be [globally unique](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id). What do you think about dynamically creating an id & `htmlFor`?

Reply:
brain fart; you're right. good catch thanks!